portafly/src/i18n/locales/en/index.ts

```diff
@@ -1,11 +1,46 @@
 // @ts-nocheck // TODO: make tslint to recognize yml files
+
+// Shared files
 import shared from 'i18n/locales/en/shared.yml'
 import overview from 'i18n/locales/en/overview.yml'
-import analytics from 'i18n/locales/en/analytics.yml'
-import applications from 'i18n/locales/en/applications.yml'
-import accounts from 'i18n/locales/en/accounts.yml'
-import integration from 'i18n/locales/en/integration.yml'
 import login from 'i18n/locales/en/login.yml'
+// Applications
+import applicationsListing from 'i18n/locales/en/applications/listing.yml'
+import applicationsPlans from 'i18n/locales/en/applications/applications_plans.yml'
+// Accounts
+import accountPersonal from 'i18n/locales/en/account_settings/personal/personal.yml'
+import accountPersonalTokens from 'i18n/locales/en/account_settings/personal/tokens.yml'
+import accountPersonalDetails from 'i18n/locales/en/account_settings/personal/personal_details.yml'
+import accountPersonalNotificationPreferences from 'i18n/locales/en/account_settings/personal/notification_preferences.yml'
+import accountUsersListing from 'i18n/locales/en/account_settings/users/listing-invitations.yml'
+// Analytics
+import analyticsUsage from 'i18n/locales/en/analytics/usage.yml'
+// Integration
+import integrationConfiguration from 'i18n/locales/en/integration/configuration.yml'
+
+const applications = {
+  listing: applicationsListing,
+  plans: applicationsPlans
+}
+
+const accounts = {
+  personal: {
```

Comment:
@didierofrivia @josemigallas 
We can have a namespace for accounts, and then an item for every page (personal, tokens, details, etc.)
Or, we can have a namespace for every page, I left this as a single namespace because that's what we talked in last meeting.
WDYT?

Reply:
Nothing against it right now, let's see how it evolves...